WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Payment Methods/ShippingLabelPaymentMethods.swift

```diff
@@ -108,6 +170,19 @@ private extension ShippingLabelPaymentMethods {
                                 + " %1$@ is a placeholder for the account display name."
                                 + " %2$@ is a placeholder for the username."
                                 + " %3$@ is a placeholder for the WordPress.com email address.")
+        static let addCreditCardButton = NSLocalizedString("Add credit card",
+                                                           comment: "Button title in the Shipping Label Payment Method screen")
+        static let addAnotherCreditCardButton = NSLocalizedString("Add another credit card",
+                                                                  comment: "Button title in the Shipping Label Payment Method" +
+                                                                    " screen if there is an existing payment method")
+        static let paymentMethodWebviewTitle = NSLocalizedString("Payment method",
+                                                            comment: "Title of the webview of adding a payment method in Shipping Labels")
+        static let doneButtonAddPayment = NSLocalizedString("Done",
+                                                            comment: "Done navigation button in Shipping Label add payment webview")
+        static let paymentMethodAddedNotice = NSLocalizedString("Payment method added",
+                                                                comment: "Notice that will be displayed after adding a new Shipping Label payment method")
+        static let pleaseAddPaymentMethodMessage = NSLocalizedString("Please, add a new payment method",
```

Comment:
Nit: Maybe keep this message simple as "No payment method found"? Since the action button already says  `Add credit card` I think this sounds better.